src/unfold/templates/unfold/helpers/userlinks.html

```diff
@@ -7,13 +7,19 @@
 
         <input type="checkbox" class="hidden peer" id="show-userlinks" />
 
-        <div class="absolute bg-white border flex-col hidden leading-none py-1 right-0 rounded-md shadow-lg text-sm text-gray-500 top-10 w-48 peer-checked:flex">
+        <div class="absolute bg-white border flex-col hidden leading-none py-1 right-0 rounded-md shadow-lg text-sm text-gray-500 top-10 min-w-[12rem] max-w-[18rem] peer-checked:flex">
             {% if nav_global %}
                 {{ nav_global }}
 
                 <div class="bg-gray-100 h-px my-1"></div>
             {% endif %}
 
+            <div class="truncate flex justify-start flex-col items-start mx-4 ">
```

Comment:
Not really an easy task. It would be nice, but Django does not have proper reverse admin URL: https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#reversing-admin-urls